drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -168,68 +301,116 @@ local function initialize_switch(driver, device)
             vendor_provided_label = name
           }
         )
+        current_component_number = current_component_number + 1
```

Comment:
current_component_number shouldn't increase when creating a child device since this PR does not support combo button/switch devices

Reply:
Fixed in 340df0f5c6c6d95d4d9b7c19e2b70b4b2fb646e2